### PR TITLE
Windows prebuilts updates with freetype and SDL_ttf 2.0.18

### LINKED ISSUE #1113: Update freetype version to 2.9.1+
I noticed that pygame is about 6 versions (and at least 5 years) behind the latest version of freetype. It looks like there have been some potentially nice changes over the last five years that may improve font rendering for pygame - mainly the addition of ClearType hinting.

I suppose there is a chance it might just be an easy copy and paste job :)

**To Do**

- [ ] Test what freetype version reported on linux with `freetype.get_version()`.
- [x] Test what freetype version reported on mac with `freetype.get_version()`.
- [x] Test what freetype version reported on windows dev10 wheel with `freetype.get_version()`.
- [ ] Resolve issue with out of date freetype library used in windows wheels.

**Related Docs:** https://www.pygame.org/docs/ref/freetype.html#pygame.freetype.get_version

### PR DESCRIPTION
**Pull request by Starbuck5**
For #3023
Also fixes #1113 
Also makes progress on #464 and #2574

This PR updates our SDL_ttf version to 2.0.18 (from 2.0.15) on Windows. This is a little more complex than usual, since SDL_ttf now is statically links to freetype. We were relying on their freetype DLL for pygame.freetype. 

So I thought this was a good opportunity to go through and modernize the old "pygame-1.9.2-prebuilt" downloads. I removed a bunch of superfluous stuff, bringing the download size down a lot, and then I added in a modern freetype DLL and headers provided by https://github.com/ubawurinna/freetype-windows-binaries

SDL_ttf 2.0.18 is exciting because it has a multiline text rendering API, it has support for color emojis, and has harfbuzz support (harfbuzz is also statically linked into the SDL_ttf lib so we don't need to worry about that).

Emojis through pygame.font:
Capture

### CODE REVIEW

File: buildconfig/config_win.py [hunk 1]
```diff
@@ -421,15 +421,8 @@ def setup_prebuilt_sdl2(prebuilt_dir):
...
-    ftDep = DEPS.add('FREETYPE', 'freetype', ['SDL2_ttf-[2-9].*', 'freetype-[1-9].*'], r'(lib)?freetype[-0-9]*\.dll$',
-                     find_header=r'ft2build\.h', find_lib=r'libfreetype[-0-9]*\.lib')
-    ftDep.path = fontDep.path
-    ftDep.inc_dir = [
-        os.path.join(prebuilt_dir, 'include').replace('\\', '/')
-    ]
-    ftDep.inc_dir.append('%s/freetype2' % ftDep.inc_dir[0])
-    ftDep.found = True
+    DEPS.add('FREETYPE', 'freetype', ['freetype'], r'freetype[-0-9]*\.dll$',
+                     find_header=r'ft2build\.h', find_lib=r'freetype[-0-9]*\.lib')
```
Comment (ankith26) [inline buildconfig/config_win.py:425]: Why is this line needed here? I believe the same thing is done a few lines above too

**Review by ankith26 — APPROVED** [review]
Overall, really nice PR! 👍 💯 

I like how the prebuilts are lighter now, and only contain the things needed. The previous prebuilts had SDL1 which gave autocomplete issues while working on pygame, so this change should fix this.

As discussed before, for the middle term it's still worth getting a new dependency system for windows, but since that can take a while to get correct, this PR improves things in the mean time

Left a review, which is more of a question :)

**Review by dr0id — APPROVED** [review]
resolve open comment, approved anyway

File: buildconfig/config_win.py (hunk 1 above)
Comment (Starbuck5) [reply buildconfig/config_win.py:425]: I'm not confident enough in the control flow of config_win to say definitely without further testing. The two methods appear to be run in different "modes," but I'm only familiar with the one mode for windows buildconfig. Regardless, I think it's safe to just change them both to point to the correct freetype.

**Comment by Starbuck5** [comment]
> As discussed before, for the middle term it's still worth getting a new dependency system for windows, but since that can take a while to get correct, this PR improves things in the mean time

Never!
I mean, I don't see the need.

All of the SDL_ deps come nicely from their website prebuilt with all of their deps
The only other things we need are freetype and portmidi

We can get freetype programmatically from https://github.com/ubawurinna/freetype-windows-binaries.
And we could do the same with portmidi if someone 😉 went to the portmidi github and implemented windows prebuilt uploading for them.
